packages/core/src/external/carequality/ihe-gateway-v2/utils.ts

```diff
@@ -11,3 +14,11 @@ export function normalizeGender(gender: "M" | "F" | undefined): "male" | "female
 export function timestampToSoapBody(createdTimestamp: string): string {
   return dayjs(createdTimestamp).toISOString();
 }
+
+export function stripCidPrefix(cid: string): string {
+  return cid.replace(cidPrefixRegex, "");
+}
+
+export function stripTags(content: string): string {
+  return content.replace(endTagRegex, "");
+}
```

Comment:
These "utils" files always get too large and with multiple diff things in them.

Can we have a dedicated file for these, please? Not sure if `xml.ts`, `markup.ts`, or something more specific to CID/SAML?